### ISSUE TITLE
De-conflict CAP data push branch and move 13777 off swirt's fork

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#16771]
## User Story or Problem Statement
The PR #13777 for pushing CAP data is blocked on lighthouse.

There are two problems:
- It has a bunch of conflicts because we have done other work on our LH Push code in the 9 months since it was created.
- It is on Swirt's fork which may vaporize when he leaves the project.

## Acceptance Criteria
- [x] Code is has all conflicts resolved
- [x] Code is moved to a PR based on branch rather than swirt's fork.
- [x] Relevant connections are made between the new PR and the old PR and the issue.
- [x] Important information from the old PR description and comments is moved to the new PR.

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [x] Facilities
- [ ] Accelerated Publishing

**Comment by swirtSJW** [comment]
This is now the new integration branch.  Deconflicted and in its own branch on the base repository. https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16837

**Comment by swirtSJW** [comment]
I added QA steps and notes about what work remains to be done once LH unblocks us.
